orchagent/routeorch.h

```diff
@@ -54,6 +55,28 @@ struct NextHopObserverEntry
     list<Observer *> observers;
 };
 
+struct RouteBulkContext
+{
+    std::deque<sai_status_t>            object_statuses;    // Bulk statuses
+    NextHopGroupKey                     tmp_next_hop;       // Temporary next hop
+    sai_object_id_t                     vrf_id;
+    IpPrefix                            ip_prefix;
+
+    RouteBulkContext()
+    {
+    }
+
+    // Disable any copy constructors
```

Comment:
No need to disable by now. My purpose here is to prevent mistakenly using non-reference in parameter list, because the main usage of this class is for in/out parameters.

There is no harm to assign. I don't have use case to support it or against it right now.

---
In reply to: [408269315](https://github.com/Azure/sonic-swss/pull/1238#discussion_r408269315) [](ancestors = 408269315)